forge-gui/res/cardsfolder/upcoming/star_athlete.txt

```diff
@@ -0,0 +1,12 @@
+Name:Star Athlete
+ManaCost:2 R R
+Types:Creature Human Warrior
+PT:3/2
+K:Menace
+K:Blitz:3 R
+T:Mode$ Attacks | ValidCard$ Card.Self | TriggerZones$ Battlefield | Execute$ DBPump | TriggerDescription$ Whenever Star Athlete attacks, choose up to one target nonland permanent. Its controller may sacrifice it. If they don't, Star Athlete deals 5 damage to that player.
+SVar:DBPump:DB$ Pump | ValidTgts$ Permanent.nonLand | RememberTargets$ True | TgtPrompt$ Select up to one target nonland permanent | SubAbility$ DBDamage
+SVar:DBDamage:DB$ DealDamage | Defined$ TargetedController | UnlessCost$ Sac<1/Permanent.nonLand+IsRemembered/targeted nonland permanent> | NumDmg$ 5 | SpellDescription$ Sacrifice targeted nonland permanent or Star Athlete deals 5 damage to controller | SubAbility$ DBCleanup

```

Comment:
the SpellDescription param does nothing here now (usually true on triggers)

Reply:
I saw you just changed that and Oracle text does not have CARDNAME. Understood on all. Thank you!